packages/components/src/components/icon/common/ic-linux.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Could you optimize these icons?

Reply:
The icons are already optimised and ran through svgo command

Reply:
It doesn't look optimized. If it is optimized it will be in a single line